### PR TITLE
Added RequestGuildMembers to request guild members from the gateway

### PR DESCRIPTION
**Pull request by jonas747**
https://discordapp.com/developers/docs/topics/gateway#gateway-request-guild-members

### CODE REVIEW

File: wsapi.go [hunk 1]
```diff
@@ -269,6 +269,44 @@ func (s *Session) UpdateStatus(idle int, game string) (err error) {
...
+
+type requestGuildMembersOp struct {
+	Op   int                     `json:"op"`
+	Data requestGuildMembersData `json:"d"`
+}
+
+// RequestGuildMembers requests guild members from the gateway
+// The gateway responds with GuildMembersChunk events
+// guildID  : the ID of the guild to request members of
+// query    : string hat username sarts with, leave empty to return all members
```
Comment (iopred) [inline wsapi.go]: Spelling, sorry for being pedantic :)

Comment (jonas747) [inline wsapi.go]: My T key is dying :'(

Comment (iopred) [inline wsapi.go]: You missed 'sarts with' :)

Comment (jonas747) [inline wsapi.go]: woops

**Pull request merged**